packages/emails/templates/attendee-scheduled-email.ts

```diff
@@ -22,14 +22,6 @@ export default class AttendeeScheduledEmail extends BaseEmail {
     this.attendee = attendee;
     this.showAttendees = showAttendees;
     this.t = attendee.language.translate;
-
-    if (!this.showAttendees) {
```

Comment:
What was this causing?

Reply:
This was hiding other attendees but just for emails. I decided to move this logic up one level so all emails and calendar integrations will receive the hidden attendee array